common/scala/src/main/scala/whisk/core/entity/ActivationId.scala

```diff
@@ -88,13 +91,23 @@ protected[core] object ActivationId extends ArgNormalizer[ActivationId] {
         def read(value: JsValue) = Try {
             val JsString(s) = value
```

Comment:
The schema dictated the activation id is a string. Will add test though and harden for JsNumber. Thanks.